management/src/test/java/org/kaazing/gateway/management/jmx/JmxRoundTripLatencyIT.java

```diff
@@ -93,7 +93,6 @@
                             .realm()
                                 .name("jmxrealm")
                                 .description("realm for jmx")
```

Comment:
Yes, I removed them because the there's no token exchange. The JmxManagementService is built to require credentials when a client is connecting to the MBeanServer. The realm is actually needed to authenticate the clients of the management service.